sdk/communication/azure-communication-phonenumbers/samples/update_phone_number_capabilities_sample.py

```diff
@@ -26,15 +26,15 @@
 
 connection_str = os.getenv('AZURE_COMMUNICATION_SERVICE_CONNECTION_STRING')
 phone_number_to_update = os.getenv(
-    "AZURE_COMMUNICATION_SERVICE_PHONE_NUMBER_TO_UPDATE" # e.g. "+15551234567"
+    "AZURE_COMMUNICATION_SERVICE_PHONE_NUMBER" # e.g. "+15551234567"
```

Comment:
I thought we were making a separate number for each language, should we not have a language specific identifier in the environment variable? Or will the same variable be used but get assigned a different number based on the language?

Reply:
We created new variables in the keyvault and changed the tests.yml file to reflect that. The environment variable still has the same name.